### PR TITLE
Allow traversing into post_thumbnail

### PR DESCRIPTION
**Pull request by jamesgol**
Related to #4719

### CODE REVIEW

**Comment by pglewis** [comment]
This works for me for the config that reproduces #4719

**Comment by pglewis** [comment]
I'd like to get @sc0ttkclark's review on this one just for one more set of eyes/sanity check.

File: classes/Pods.php [hunk 1]
```diff
@@ -1691,6 +1691,9 @@ public function field( $name, $single = null, $raw = false ) {
 											} else {
 												$value[] = $item->{$field};
 											}
+										} elseif ( 0 === strpos( $full_field, 'post_thumbnail' ) ) {
```
Comment (sc0ttkclark) [inline classes/Pods.php]: We need to check if `! empty( $related_obj )` here too.

**Comment by pglewis** [comment]
Done, needs @sc0ttkclark clearance

**Comment by pglewis** [comment]
Changelog: Single-select Post Thumbnail does not display in Pods Template traversal (#4719)

**Pull request merged**